src/declarations/stencil-public-compiler.ts

```diff
@@ -1993,6 +2009,12 @@ export interface OutputTargetDistCustomElements extends OutputTargetBaseNext {
    * Enables the generation of type definition files for the output target.
    */
   generateTypeDeclarations?: boolean;
+  /**
+   * Define the export/definition behavior for the output target's generated output.
+   * This controls if/how custom elements will be defined or where components will be exported from.
+   * If omitted, no auto-definition behavior or re-exporting will happen.
+   */
+  customElementsExportBehavior?: CustomElementsExportBehavior | null;
```

Comment:
Yeah I think that can be removed since I made a change to add validation logic to the target so that after validation runs there is always a value assigned and it defaults to `DEFAULT`